addons/mail/models/discuss/res_partner.py

```diff
@@ -18,6 +18,16 @@ class ResPartner(models.Model):
         copy=False,
     )
     channel_member_ids = fields.One2many("discuss.channel.member", "partner_id")
+    is_in_call = fields.Boolean(compute="_compute_is_in_call")
+
+    @api.depends("channel_member_ids.rtc_session_ids")
+    def _compute_is_in_call(self):
+        partner_ids_in_call = self.env["discuss.channel.member"]._read_group(
+            [("partner_id", "in", self.ids), ("rtc_session_ids", "!=", False)],
+            aggregates=["partner_id:array_agg"],
+        )[0][0]
```

Comment:
Can't we just define the inverse of partner_id on rtc session to compute this trivially through fields (existence of the inverse) instead of a systematic query?